### DISCUSSION TITLE
Duda sobre commit en repositorio remoto

### DISCUSSION THREAD

**Discussion by Lautymendez** [INGCOM-UNRN-P1/cursada-2023#32] [Preguntas y respuestas] [answered]
Buenass, acabo de hacer un commit y me di cuenta que estaba mal, el problema es que era tarde porque ya se subio al repositorio remoto. ¿Como puedo eliminar un commit que ya subi al repositorio remoto?

**Comment by JeanFranco99** [comment]
Tenes que hacer un 
git revert _N° commit_
El n° de commit lo encontras en el repositorio privado, haciendo click aca 
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/c414967b-d59d-4df7-9e28-a8b516d60578)
Una vez que entras al codigo
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/2069c6b9-dde8-4e4c-8e9f-5b254913a43d)
Copias este y listo
git commit revert 2193de5

**Reply by martinvilu** [reply]
Manipular la historia no es una buena idea, modifica el archivo y que siga su historia.

**Reply by Lautymendez** [reply]
Gracias igual Jean!

**Comment by martinvilu** [comment, marked as answer]
Por qué está mal?

La idea de los repositorios es ver cómo evolucionan los archivos que lo contienen

Al corregir, por otro lado, solo tomo el archivo en su último estado.

No hay razón para andar metiendo mano y en especial, perder tiempo con dejar la historia _perfecta_.

Mira los commits de este mismo repositorio para más ejemplos

**Reply by Lautymendez** [reply]
Aaa, claro pense que estaba mal porque copie el archivo del ejercicio 1 (TP 1) y luego hice un commit del ejercicio 2 con el mismo codigo que contenia el ejercicio 1 . 
Pero ahora entendi que es la historia de como se creo el codigo y no tiene tanta relevancia; lo peor es que perdi tiempo 😫, pero al menos estoy aprendiendo cada vez mas.
Gracias martin!

**Reply by martinvilu** [reply]
> pero al menos estoy aprendiendo cada vez mas

Esto es lo importante
